pom.xml

```diff
@@ -30,10 +30,10 @@ under the License.
 
     <groupId>org.apache.flink</groupId>
     <artifactId>flink-shaded</artifactId>
-    <version>1</version>
+    <version>1.0</version>
```

Comment:
It is more likely that we have a bugfix for a child, that's true. I felt that it would be more consistent if the "primary" version is consistent across parent and children.